app/views/layouts/application.html.erb

```diff
@@ -64,14 +64,14 @@
   </head>
 
   <body class=<%= show_application_background? ? 'with-background' : ''%>>
-    <div class="container hero is-fullheight">
-      <nav class="navbar is-spaced has-shadow is-navbar-budgeting-kid-color"
-           role="navigation"
-           aria-label="main navigation">
+    <nav class="navbar wrapper is-navbar-budgeting-kid-color <%= navbar_hide? ? 'hidden' : ''%>"
```

Comment:
add please `is-radiusless` class

Reply:
@VladislavSokov, could you please clarify your statement, for which element and why?

Reply:
It's a feature. Ok, if you'd like.